public/usage-examples/physics/scale_matrix_from_vector/scale_matrix_from_vector-simple.cpp

```diff
@@ -0,0 +1,26 @@
+#include "splashkit.h"
+
+int main()
+{
+    // Define the scale factors
+    vector_2d matrix_scale;
+    matrix_scale.x = 2.0; // Scale width by 2
+    matrix_scale.y = 1.0; // Keep height unchanged
+
+    // Create a scaling matrix using the scale factors
+    matrix_2d scaling_matrix = scale_matrix(matrix_scale);
+
+    // Print the scaling matrix
```

Comment:
this comment is different from other files. should be "Print the scaling matrix to the console" for consistency